docs/Containerfile.5.md

```diff
@@ -154,6 +154,49 @@ Current supported mount TYPES are bind, cache, secret and tmpfs.
 
               · rw, read-write: allows writes on the mount.
 
+**RUN --network**
+
+> Added in https://github.com/containers/buildah/pull/4566
+
+`RUN --network` allows control over which networking environment the command
+is run in.
+
+Syntax: `--network=<TYPE>`
+
+**Network types**
+
+| Type                                         | Description                            |
+|----------------------------------------------|----------------------------------------|
+| [`default`](#run---networkdefault) (default) | Run in the default network.            |
```

Comment:
The syntax confuses me. Is it really `networkdefault` and not `network=default`?

Reply:
buildah syntax is `network=default` the text `#run---networkdefault` is markdown link to the subheading, if you render the markdown it should allow users to click on options which should automatically scroll to correct subheading. Text `#run---networkdefault` is not visible on rendered markdown.